integration-authentication-api/pom.xml

```diff
@@ -16,6 +16,18 @@
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
```

Comment:
Yes because I had to create the UnixTimestampDeserializer, which uses this dependency. I have tried to move it to the commons package but a circular reference would be made in that case. AN alternative would be to remove the Deserializer, change the attribute type to long and create an getExpirationDate() that adds "000" in the end of this long number and return a new java.util.Date().